plugins/DebuggerCore/unix/linux/PlatformThread.cpp

```diff
@@ -87,6 +87,7 @@ int PlatformThread::priority() const  {
 // Desc:
 //------------------------------------------------------------------------------
 edb::address_t PlatformThread::instruction_pointer() const  {
+	// FIXME(ARM): doesn't work at least on ARM32
```

Comment:
On my kernel version this is unreliable for x86-64 as well. I don't know which version broke this, but it also seemed not super critical at the time since I only noticed it being used by the thread state display (not for things like the register view and such).